examples/example-controllers/tests/helpers.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+import type { RestrictedControllerMessenger } from '@metamask/base-controller';
```

Comment:
These helpers are already defined in `packages/base-controller/tests/helper.ts`, but for some reason I couldn't reuse them properly. I found that if I passed either `PetNamesControllerMessenger` or `GasPricesControllerMessenger`, I was consistently getting `never`. However if I copy those helpers to `packages/example-controllers`, then they return the right types. I'll have to investigate this further.